sporkfish/lichess_bot/lichess_bot_berserk.py

```diff
@@ -144,13 +145,39 @@ def set_pos_and_play_move(
                 num_moves = len(state["moves"].split())
                 prev_moves = state["moves"]
                 set_pos_and_play_move(num_moves, color, prev_moves, game_id, state)
+            elif state["type"] == "opponentGone" and state["gone"]:
+                time.sleep(8)
+                self.client.board.claim_victory(game_id)
```

Comment:
Could you add a line
`return` 
after this one? If I read this correctly,  you will claim victory and we go back to to the for loop. However since the game has ended there won't be any new states, so you'll just be forever waiting there. I know there's already an existing hanging issue with the previous code but since this is new code, might as well fix it while you add it. We have stricter gatekeeping for code now :)

Reply:
Wait, I just read the API. https://lichess-org.github.io/berserk/api.html#berserk.clients.Board This is for external (physical electronic boards). Have you tested?

Reply:
Wait. On the API https://lichess.org/api#tag/Teams/operation/teamIdPmAll it says:

> Play on Lichess with physical boards and third-party clients. Works with normal Lichess accounts. Engine play or assistance is [forbidden](https://lichess.org/page/fair-play).

I don't see any mention of this working for bot accounts. In fact engine play is forbidden. Have you tested this works?

Reply:
let me look into how this works more. in fact, i tried abandoining the game and being afk but it didnt trigger game state opponent gone